tools/console_version

```diff
@@ -1,2 +1,2 @@
-COMMIT="1d31f285ca035737b9d7b04f4e4efa6baef9ca25"
-SHA2="2390e06f21a70463742129fbd4deee0e091140a5a9f28a10eb72a5d988787d1f"
+COMMIT="056d455a33e0c1a2603d7d704e0c6afc8f5b1937"
+SHA2="<?xml version='1.0' encoding='UTF-8'?><Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message></Error>"
```

Comment:
dunce cap lmao. need to fix my console version-bumper script to fail on the 404 that produced this foolishness